person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
```

Comment:
- [ ] Kindly make sure to add attr_reader and attr_accessor before initialize methods.

Reply:
Hi, @sja-thedude !!!
Thanks for the review. Is there a particular reason for adding the readers and accessors before the initialize method, please?